html/webappapis/structured-clone/structured-clone-battery-of-tests.js

```diff
@@ -79,6 +80,8 @@ check('primitive number, 9007199254740992', 9007199254740992, compare_primitive)
 check('primitive number, -9007199254740992', -9007199254740992, compare_primitive);
 check('primitive number, 9007199254740994', 9007199254740994, compare_primitive);
 check('primitive number, -9007199254740994', -9007199254740994, compare_primitive);
+check('primitive BigInt, 1n', 1n, compare_primitive);
+check('primitive BigInt, 1000000000000000000000000n', 1000000000000000000000000n, compare_primitive);
```

Comment:
Do enough browsers implement this syntax now that it won't syntax-error the whole test?

Reply:
Looks like Safari doesn't, hrm. https://wpt.fyi/results/workers/semantics/structured-clone/dedicated.html?diff&filter=ADC&run_id=325050012&run_id=330640001

Reply:
Oh wow, not even on TP.

@caiolima thoughts?

Reply:
Are there other tests that use this literal that are also failing? Filing a WebKit but would be nice.

Reply:
BigInt is not feature complete in WebKit yet. It is available behind the `useBigInt` runtime flag, but support for `++`, `—` and TyepdArrays is missing.

Reply:
@caiolima would you happen to know how to enable that in Safari Technology Preview, and would that be a good idea to do?

Reply:
Into macOS it is `__XPC_JSC_useBigInt=true open -a 'Safari Technology Preview'`. Since it is not supported into `++`, `--` and TypedArrays, I would expect some exceptions being throw or even some crashes.